recipes/pfapack/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pfapack" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c415dc8723ab9b5d96cda21b0fa115a36d6d1e758fc2c4943c2952b958dca81a
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This can be noarch python

Reply:
Thanks! 

Is this https://github.com/conda-forge/staged-recipes/pull/10970/commits/b08bf9c679e860343132c84ab01b5b511e54baa0 sufficient?

Reply:
Actually, I need `skip: True  # [py<36]` and this package doesn't work on Python 2.

Reply:
You can use `python >=3.6` in the requirements instead

Reply:
Done! 👍